PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -988,6 +1002,10 @@ BootOsImage (
     ClosePartitions (HwPartHandle);
   }
 
+DEBUG_CODE_BEGIN();
```

Comment:
Since the code will go through now, better to update label at line #992 to  "Exit:" instead of "Error".
Also change the comment at line #989.

Reply:
Updated the label with "Exit" and removed comment "Never reach here". Thanks for the comment.